docs/lang/articles/basic/external.md

```diff
@@ -227,3 +227,31 @@ copy(x, y) # error!
 copy(x, y.clone()) # correct
 copy(x, y.contiguous()) # correct
 ```
+
+## FAQ
+
+### Can I use `@ti.kernel` to accelerate a NumPy function?
+
+You may try to call NumPy functions to process an array inside the Taichi scope, as the following example shows:
+
+```python
+import numpy as np
+
+@ti.kernel
+def invalid_sum(arr: ti.types.ndarray()):
+    total = np.sum(arr)
+    ...
```

Comment:
We should put the error message in the comment so that it's obvious non-supported case.